.github/workflows/docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy docs to GitHub pages
+
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  generate-and-deploy-docs:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Generate docs
+        run: |
+          cargo doc --workspace --all-features --release --no-deps --document-private-items --target-dir ./generated_docs
+          cd generated_docs/doc
+          echo '<meta http-equiv="refresh" content="0; url=https://galacticcouncil.github.io/Basilisk-node/basilisk/index.html">' > index.html
```

Comment:
redirect should be relative, not absolute

have you tried to just deploy `basilisk` subdirectory? that would remove need for redirect all together

Reply:
`basilisk` subdirectory is just for the documentation of the basilisk binary. All subdirectories are required. Each target has its own subdirectory.